lib/metrix.ex

```diff
@@ -22,6 +24,18 @@ defmodule Metrix do
     end
   end
 
+  defp initial_modifiers do
```

Comment:
What's the purpose of this pass-through function? Seems like we only need one function to get the initial modifiers.